webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/IsosurfaceService.scala

```diff
@@ -174,7 +173,7 @@ class IsosurfaceService @Inject()(
     val cuboid = request.cuboid
     val voxelDimensions = Vector3D(request.voxelDimensions.x, request.voxelDimensions.y, request.voxelDimensions.z)
 
-    val dataRequest = DataServiceDataRequest(request.dataSource,
+    val dataRequest = DataServiceDataRequest(request.dataSource.orNull,
```

Comment:
What is the orNull for? Seems dangerous. Are there null checks in place?

Reply:
Yes, that's how we create `DataServiceDataRequest`s in the `VolumeTracingService` (l. 213) as well.

Reply:
maybe add a comment at DataServiceDataRequest ` //null in volume tracings. is never read there.` or something like that